### ISSUE TITLE
Cannot read property 'msie' of undefined

### ISSUE THREAD

**Issue by leckylao** [cowboy/jquery-hashchange#43]
Uncaught TypeError: Cannot read property 'msie' of undefined jquery.ba-hashchange.min.js?body=1:10

Uncaught TypeError: Cannot read property 'msie' of undefined jquery.ba-hashchange.js?body=1:300

 $.browser.msie && !supports_onhashchange && (function(){

Seems like the msie is undefined. And I do need support for IE8.

**Comment by rohan-deshpande** [comment]
Yeah I'm seeing it as well, would be due to jQuery removing $.browser I guess

**Comment by Krinkle** [comment]
Duplicate of #42.

**Comment by rafegoldberg** [comment]
seeing this problem as well, any suggestions on a fix? and :+1: @rohan-deshpande, I'll bet that's the issue.
